### PR TITLE
Added db.properties

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

**Review by fyarkiy — APPROVED** [review]
All is good

File: src/main/java/spring/intro/config/AppConfig.java [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
-import javax.sql.DataSource;
 import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
 import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
 import spring.intro.model.User;
```
Comment (Sofasmile) [inline src/main/java/spring/intro/config/AppConfig.java:14]: run `mvn clean package` -> check style will be failed

File: src/main/java/spring/intro/config/AppConfig.java [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
+
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.service",
         "spring.intro.dao"
 })
 public class AppConfig {
+    @Autowired
+    private Environment env;
```
Comment (Sofasmile) [inline src/main/java/spring/intro/config/AppConfig.java]: What is better: setter, constructor, or field injection?

Comment (Sofasmile) [inline src/main/java/spring/intro/config/AppConfig.java]:
```suggestion
    private Environment environment;
```

Comment (Nokafan) [reply src/main/java/spring/intro/config/AppConfig.java]: Constructor is the best.

Comment (Nokafan) [reply src/main/java/spring/intro/config/AppConfig.java]: Fixed

File: src/main/java/spring/intro/config/AppConfig.java (hunk 1 above)
Comment (Nokafan) [reply src/main/java/spring/intro/config/AppConfig.java:14]: не фейлиться (

Comment (Nokafan) [reply src/main/java/spring/intro/config/AppConfig.java:14]: Пофіксив чекстайл

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**